src/core.js

```diff
@@ -46,7 +46,15 @@ let Game = {
    *     },
    *     guesses: 1                 // Not present if no prompt set
    *   },
-   *   winner : 'red|blue'          // Not present if neither team has won yet
```

Comment:
main reason is probably idk how js classes work haha - that sounds like a good idea